<launch>   
    <arg name="default_utm_zone" default="33N" />

    <rosparam file="$(find cuzk_tools)/config/geomap_colours.yaml" />

    <node pkg="tf2_ros" type="static_transform_publisher" name="static_tf_fake" args="428333 5539106 0 0 0 0 utm utm_local" />
    <node pkg="tf2_ros" type="static_transform_publisher" name="base_link_utm_local_fake" args="42202 -29073 0 0 0 0 utm_local base_link_fake" />

    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find cuzk_tools)/rviz/elev_topo.rviz" output="screen">
    </node>
    <!-- <node pkg="rviz" type="rviz" name="rviz" args="-d $(find cuzk_tools)/rviz/new_topography_vis.rviz" output="screen">
    </node> -->

    <node pkg="cuzk_tools" type="elevation" name="elevation" output="screen">
        <param name="default_utm_zone" type="string" value="$(arg default_utm_zone)"/>
    </node>

    <node pkg="cuzk_tools" type="pose_elevation_srv" name="pose_elevation_srv" output="screen">
        <param name="default_utm_zone" type="string" value="$(arg default_utm_zone)"/>
    </node>

    <node pkg="cuzk_tools" type="pose_elevation_tf" name="pose_elevation_tf" output="screen">
        <param name="default_utm_zone" type="string" value="$(arg default_utm_zone)"/>
        <param name="default_followed_tf" type="string" value="base_link"/>
    </node>

    <node pkg="cuzk_tools" type="orto2mesh" name="orto2mesh" output="screen">
        <param name="pcd_topic" type="string" value="elevation_utm_local"/>
        <param name="default_utm_zone" type="string" value="$(arg default_utm_zone)"/>
    </node>


    <node pkg="cuzk_tools" type="topography" name="topography" output="screen">    
        <param name="default_utm_zone" type="string" value="$(arg default_utm_zone)"/>
    </node>


    <node pkg="cuzk_tools" type="viz_map" name="viz_map" output="screen">
    </node>
</launch>    

<!-- overit offline, script na download, ukladat do /home/.ros/cuzk_data, vizualizace - yaml config, jak obarvovat;-->